content/03.categorize.md

```diff
@@ -40,6 +40,13 @@ For the melanoma task, reported performance was competitive with or better than
 
 Reusing features from natural images is also an emerging approach for radiographic images, where datasets are often too small to train large deep neural networks without these techniques [@tag:Bar2015_nonmed_tl; @tag:Shin2016_cad_tl;
```

Comment:
Should we move the `Reusing features` line to follow the `An alternative in the domain` line and lead with `Data augmentation is a strategy...`?  This edited version splits the discussion of natural images.